www/participate/donate.html

```diff
@@ -9,16 +9,24 @@ <h3>Freifunk in Berlin durch Spenden unterstützen!</h3>
 
 <p>
   Für den Aufbau und Erhalt unseres freien Funknetzwerks sind wird auf
-  regelmäßige Spenden angewiesen. Alle Spenden werden vom <a
-    href="http://foerderverein.freie-netzwerke.de/" class="externalLink">
-  Förderverein freie Netzwerke e.V.</a> bearbeitet. Der Verein bevorzugt Spenden über
-  Betterplace. Zuwendungsbescheide können vom Verein ausgestellt werden.
+  regelmäßige Spenden angewiesen. Wir bevorzugen Spenden über betterplace.org. Das ist
+  sowohl für uns, als auch für euch einfacher. Spendenbescheinigungen werden von betterplace.org 
+  ausgestellt und im Februar des Folgejahres verschickt.
+  <br><br>
   Ihr könnt Freifunk Berlin auf verschiedenen Wegen eure Geld-Spende zukommen
   lassen:
 </p>
 
 <h4>Betterplace:</h4>
 
+<iframe frameborder="0" marginheight="0" marginwidth="0"
+  src="https://www.betterplace-widget.org/projects/83703?l=de"
+  width="100%" height="320" style="border: 0; padding:0; margin:0;"
+  >Informieren und spenden: <a href='https://www.betterplace.org/de/projects/83703-freifunk-berlin-im-haus-der-statistik'
+  target='_blank'>„Freifunk Berlin im Haus der Statistik“</a> auf betterplace.org öffnen.
+</iframe>
+
+<!--
```

Comment:
As the page is under version cotrol, there is no need to carry "dead code" around.
As the commented out code is not working anyway, we should just drop it.